crates/ruff/src/rules/flake8_django/rules/helpers.rs

```diff
@@ -23,6 +23,18 @@ pub fn is_model_form(checker: &Checker, base: &Expr) -> bool {
         })
 }
 
+/// Return `true` if the expression is constructor for a Django model field.
+pub fn is_model_field(checker: &Checker, expr: &Expr) -> bool {
```

Comment:
By "all of these" do you mean all of flake8-django rules or just this one?

Reply:
I was referring to all of the utility functions in this file, since they follow a similar pattern, but it's also totally fine if you only want to change this one function.

Reply:
Oh wait, I didn't see which line you were referring to.

Reply:
Thanks!